py/STREAM/Settings/EquationSystem.py

```diff
@@ -18,5 +19,6 @@ def __init__(self, settings):
         # Overwrite with STREAM-modified unknowns
         self.addUnknown('n_i', Ions(settings=settings))
         self.addUnknown('E_field', ElectricField(settings=settings))
+        self.addUnknown('tau_perp', ConfinementTime(settings=settings)) #modified
```

Comment:
no need to add comment...

Reply:
version control should keep track of this and the comment is deprecated when we merge

Reply:
The comment has been removed